sources/packages/web/src/views/institution/DynamicInstitutionProfile.vue

```diff
@@ -8,11 +8,19 @@
   <Card class="p-m-4">
     <template #content>
       <formio
+        v-if="editMode"
         formName="institutionprofile"
         :data="initialData"
         @loaded="formLoaded"
         @submitted="submitted"
       ></formio>
+      <formio
+        v-if="!editMode"
+        formName="institutionprofilecreation"
```

Comment:
can we add something like this `:formName="editMode ? 'institutionprofile' : 'institutionprofilecreation'"` and reduce LOC

Reply:
This is exactly how I initially coded but formio is unable to read expressions when they are written in param(s). I got a 400 error.

Reply:
@dheepak-aot , i checkout to this branch in my local and tried its working for me

Reply:
Please consider reducing the amount of code as @ann-aot suggested.
I would recommend creating a computed value to return the form name with the logic that Ann proposed inside. Both ways work and I would be good either way 😄

Reply:
@andrewsignori-aot Yes but as I mentioned, expression was throwing error so I stepped back.  I will have  screen share with @ann-aot and see what's going wrong at my side.

Reply:
It is fixed now. It was missing ":" before the param name.

Reply:
@ann-aot param modified with expression